src/actions/home/index.cr

```diff
@@ -1,5 +1,11 @@
 class Home::Index < BrowserAction
   get "/" do
-    render LuckyWeb::WelcomePage
+    wod = WodQuery.new.date(Time.now.at_beginning_of_day).first?
```

Comment:
`first?` means this type is `Wod | Nil`. It's cool that the if statement below forces the type to be `Wod` inside the true branch.

Reply:
Yeah it's really slick!